controllers/v2/programsController.js

```diff
@@ -0,0 +1,85 @@
+/**
+ * name : programsController.js
+ * author : Aman
+ * created-date : 02-Jun-2020
+ * Description : Programs related information
+ */
+
+// Dependencies
+const programsHelper = require(MODULES_BASE_PATH + "/programs/helper");
+const v1Programs = require(ROOT_PATH + "/controllers/v1/programsController");
+
+/**
+    * Programs
+    * @class
+*/
+
+module.exports = class Programs extends v1Programs {
```

Comment:
@aman-tunerlabs - The same API can be in usersController with endpoint as privatePrograms